docs/Authentication.md

```diff
@@ -315,3 +315,72 @@ export default withRouter(MyPage);
 {% endraw %}
 
 The `<Authenticated>` component calls the `authClient` function with `AUTH_CHECK` and `authParams`. If the response is a fulfilled promise, the child component is rendered. If the response is a rejected promise, `<Authenticated>` redirects to the login form. Upon successful login, the user is redirected to the initial location (that's why it's necessary to get the location from the router).
+
+
+## Redirect after logout
+
+By default react-admin will redirect to '/login'. There are a couple of ways to change this: 
+
```

Comment:
You say there are a couple ways to do it, but the reader won't know what's the right one. Don't give them the choice, and document only the right way (the second one IMO).